apps/docs/src/app/platform/component-docs/platform-dynamic-page/platform-dynamic-page-examples/platform-dynamic-page-tabbed-example.component.scss

```diff
@@ -0,0 +1,4 @@
+.scrollable-container {
+    height: 20rem;
+    overflow-y: scroll;
+}
```

Comment:
new line required. Please have look generic issues and update it

Reply:
although lint passes, have modified accordingly.